### PR TITLE
Logo header Navitia Explorer

### PR DESCRIPTION
**Pull request by dvdn**
Adding a logo in header
removing useless img and html portions

### CODE REVIEW

File: assets/css/global.css [hunk 1]
```diff
@@ -59,6 +59,10 @@ ul li a:hover {
     border-width:1px;
 }
 
+img.navitia-explorer{
+    height:60px;
```
Comment (nlehuby) [inline assets/css/global.css]: I find it too big :(

File: coverage.html [hunk 2]
```diff
@@ -6,6 +6,9 @@
     <link rel="stylesheet" href="./assets/easy-button/easy-button.css" />
 </head>
 <body onload="coverage_onLoad(); includeHTML();">
+    <a href="index.html" title="Go to Homepage">
```
Comment (nlehuby) [inline coverage.html]: You may want to include this in the menu.js file, which make the header of all the html files.

**Comment by nlehuby** [comment]
test with the logo as a menu item : 
nav-explo

**Comment by dvdn** [comment]
Open discussion :yum:  but i feel it's a header logo not a menu item (but inclusion can be next step).
Inspired by https://canaltp.github.io/navitia-playground/ logo/size to keep homogeneity.

File: coverage.html (hunk 2 above)
Comment (prhod) [inline coverage.html]: +1 with the comment, it's the purpose of the JS menu, to avoid the modification of all the HTML pages.

**Comment by dvdn** [comment]
Maybe next time :P

**Pull request closed without merging**